lib/shared/bucketing-assembly-script/assembly/types/dvcUserPB.ts

```diff
@@ -0,0 +1,100 @@
+import { DVCUser_PB } from './protobuf-generated/DVCUser_PB'
+import { NullableString } from './protobuf-generated/NullableString'
+import { NullableDouble } from './protobuf-generated/NullableDouble'
+import { CustomDataValue } from './protobuf-generated/CustomDataValue'
+import { _getPlatformData } from '../managers/platformDataManager'
+import { NullableCustomData } from './protobuf-generated/NullableCustomData'
+import { CustomDataType } from './protobuf-generated/CustomDataType'
+
+export class CustomDataValueInterpreter {
```

Comment:
this basically mimics the methods we had on the JSON CustomData class. They are static because it seems i can't extend the CustomDataValue PB class to add more methods to it and then cast one to the other...I'd have to copy the PB class contents to a new instance of my extended class. That results in a bunch of extra allocations we don't need.